src/main/java/com/teragrep/pth10/steps/teragrep/bloomfilter/FilterTypes.java

```diff
@@ -110,6 +108,7 @@ public Map<Long, Long> bitSizeMap() {
 
     public String pattern() {
         final String pattern;
+        String BLOOM_PATTERN_CONFIG_ITEM = "dpl.pth_06.bloom.pattern";
```

Comment:
these could be final, or better yet, object of their own

Reply:
made final. Object of some sorts would indeed be better but chose not to implement in this PR scope